requirements.txt

```diff
@@ -3,17 +3,21 @@ dnspython==2.2.1
 fonttools==4.32.0
 kiwisolver==1.4.2
 matplotlib==3.5.1
+networkx==2.8.7
 numpy==1.22.3
 packaging==21.3
 pandas==1.4.2
 Pillow==9.1.0
+plotly==5.10.0
 psutil==5.9.0
 pymongo==4.1.1
 pyparsing==3.0.8
 PyQt5==5.15.6
 PyQt5-Qt5==5.15.2
 PyQt5-sip==12.9.1
 PyQt5-stubs==5.15.2.0
+PyQtWebEngine==5.15.6
+PyQtWebEngine-Qt5==5.15.2
```

Comment:
When I run pip install -r requirements.txt I have dependency conflict between this and PyQt-sip. Is it only my local problem or you faced something similar?

Reply:
I didn't have that issue, but I've checked and I have PyQt5-sip in version 12.11.0 installed. don't know how that happened, maybe it upgraded it automatically (?). Which versions ended up working for you?